index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="eng">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <Title>Portfolio</Title>
+        <link rel="stylesheet" type="text/css" href="index.css">
+    </head>
+
+    <body>
+      <div class="header">
+        <h1 class="my-logo">My Logo</h1>
+        <a class="menu-link" href="">
+          <div class="lines">
+            <div class="menu-1"></div>
+            <div class="menu-2"></div>
+            <div class="menu-3"></div>
+          </div>
+        </a>
+      </div>
+      
+      <div class="headline">
```

Comment:
As this is a section in your application, we can use a `section` tag instead a `div` tag

- [ ] Kindly replace this `div` tag with a `section` tag to follow best practices